### DISCUSSION TITLE
online_photos & street_level_imagery

### DISCUSSION THREAD

**Discussion by ZeljkoBG** [osmandapp/OsmAnd#20885] [Q&A]
street_level_imagery - (string) Street-level imagery - is "**Mapillary**". 
It seems to me that the logical name for the string is some word/sentence where Maplillary is mentioned

what then?
online_photos - (string) Online photos  - pictures are from "**Mapillary**".  /// (Just to add that the app displays the same image twice)

What is the difference, what is it about,... 
rg

**Comment by dmpr0** [comment]
Currently Street level images are shown only from Mapillary. Source for Online photos is Wikimedia, previous both showed in one group.
